gap/perm.gi

```diff
@@ -412,7 +412,7 @@ AddMethod(FindHomDbPerm, FindHomMethodsPerm.Imprimitive,
           "for a imprimitive permutation group, restricts to block system");
 AddMethod(FindHomDbPerm, FindHomMethodsPerm.SnkSetswrSr,
           60, "SnkSetswrSr",
```

Comment:
So, what does that stand for? Based on what people told me here, the guess would be `\Sym{n over k) \wreath \Sym(r)`.  But that name still is unpronounceable, and that's bad as it inhibits communication on this code. I also don't see why the `Set` needs to be in there -- surely `SnkwrSr` says it, too?

But actually, how about we just rename the method to "LargeBasePerm" or even just "Jeyllyfish"? Both have the great advantage that they are pronounceable. (Of course it's not a requirement for this PR! I just thought you might have some thoughts on this, as you have looked at the code recently).

Reply:
I actually just now thought about how to rename it. My suggestion is `LargeBasePrimitivePerm`.

I'll do the renaming in a different, later PR because I don't want to mess with other people's PRs.

Reply:
I am pretty sure that's not the way you should do it; those formulas are meant to adhere to LaTeX conventions, so you should be using `\{` and `\}` -- see also https://www.gap-system.org/Manuals/pkg/GAPDoc-1.6.1/doc/chap3.html#X7ABF42328467E966

Reply:
OK

Reply:
Although I don't see which PR you are afraid of conflicting with?

Reply:
Ah, I wanted to do the renaming as suggested [in this hack.md](https://hackmd.io/TJflWtgcSFmSd2WHwneMtA) all at once. That's what I meant with "in a later PR".

Reply:
I really don't see a value in "doing it all at once". To the contrary: It means that changes which might be uncontroversial would be delayed by other changes that *are* controversial.

Renaming this method is uncontroversial, clearly fits into this PR, and is trivial to exclude from "that HackMD". So you could just have done it here... As it is, I'll do it myself then.